GoogleTestNuGet/Build.ps1

```diff
@@ -134,6 +134,18 @@ function Add-Signing {
     $MicroBuildProps.SetAttribute("Project", "$microbuild\build\Microsoft.VisualStudioEng.MicroBuild.Core.props")
     $MicroBuildProps.SetAttribute("Condition", "Exists('$microbuild\build\Microsoft.VisualStudioEng.MicroBuild.Core.props')")
 
+    $BuildGroup = $xml.CreateElement("ItemDefinitionGroup", "http://schemas.microsoft.com/developer/msbuild/2003")
+    $ClCompile = $xml.CreateElement("ClCompile", "http://schemas.microsoft.com/developer/msbuild/2003")
+    $AdditionalOptions = $xml.CreateElement("Profile", "http://schemas.microsoft.com/developer/msbuild/2003")
```

Comment:
I believe this should say "AdditionalOptions" instead of "Profile", I think this was my mistake originally although it seems to have picked up the setting somewhere in the latest builds. I'd suggest fixing it here to make sure it doesn't mess with anything in the future.

Reply:
Will resolve in future small PR